framework/bootstrap/configcontroller.go

```diff
@@ -553,20 +551,11 @@ func startXdsMonitorController(mc *monitorController, configRevision string, xds
 		},
 	})
 
-	if err = xdsMCP.Run(); err != nil {
-		return fmt.Errorf("failed to init client for xds config source %s, err: %+v", mc.configSource.Address, err)
-	}
-
-	// check xds cache synced or not
-	log.Infof("syncing data of xds config source [%s]", mc.configSource.Address)
-	for {
-		if xdsMCP.HasSynced() {
-			break
+	go func() {
+		err := xdsMCP.Run()
+		if err != nil {
```

Comment:
In fact, it does not return any errors

Reply:
This is not something to consider as a function caller